app/templates/auth/host/location.hbs

```diff
@@ -29,7 +29,7 @@ as |page|
         {{profile-page/transit changeset=this.changeset}}
       {{/form.group}}
 
-      {{#form.group label="Neighborhood Environment" as |group|}}
+      {{#form.group label="What are the amenities/stores nearby?" as |group|}}
```

Comment:
And this should probably be "What amenities/stores are nearby?"

Reply:
Thanks for correction!